### DISCUSSION TITLE
appleskin

### DISCUSSION THREAD

**Discussion by Quezler** [EnigmaticaModpacks/CreateTogether#9] [Ideas]
the mod that visualizes saturation: https://www.curseforge.com/minecraft/mc-mods/appleskin

![Screen Shot 2021-05-29 at 10 47 20](https://user-images.githubusercontent.com/3179271/120064183-52e97f80-c06b-11eb-803c-58687de995fb.png)

personally i first tried looking up the saturation values in the code, but those are floats and my brain can't really wrap around to the exact conversion logic behind them, and the floats of vanilla food items are hard to come by, this mod just makes it easier to compare. (mainly tried to figure out how well honeyed apples compared to golden carrots, and there are quite close)

i just think its neat to give some extra insight to players when they decide which foods to make, but that's just me :3

(of course now the hunger points are visualized twice, haven't looked into which mod does that, but its just a thought*)

**Comment by NielsPilgaard** [comment]
Good idea, I'll get that added :)

**Comment by Quezler** [comment]
@NillerMedDild would you happen to know which mod added the hunger steaks under the item name shown in the screenshot? (i doubt applecore did that but I haven't tested it tbh, just hoping its not & you might know which it is) 😅

**Reply by NielsPilgaard** [reply]
It's Quark, and I've disabled it for the next update :)
